CHANGELOG.asciidoc

```diff
@@ -30,6 +30,7 @@ endif::[]
 ===== Features
 * Add support for log correlation for `java.util.logging` (JUL) - {pull}2724[#2724]
 * Add support for spring-kafka batch listeners - {pull}2815[#2815]
+* Improved instrumentation for legacy Apache HttpClient (when not using an `HttpUriRequest`, such as `BasicHttpRequest`)
```

Comment:
I think it's actually `an` in this case but not 100% sure. IINM, for acronyms like HTTP, it depends on how they're pronounced. I guess there are different ways to pronounce HTTP but the one I usually hear people saying is `age-tee-tee-pee`. As the `a` in `age` is a vocal, it would be `an HTTP request`. However, some people also pronounce it `hage-tee-tee-pee`. In this case, it would be `a HTTP request`.

@bmorelli25 can probably give us a definite answer on that.

Reply:
I guess today I've learned something new 👍

Reply:
Oh, this is a fun one. So Felix is right—with initialisms and acronyms, it's not about how they're spelled; it's about how they're pronounced.

Quick side note: HTTP is actually an initialism—not an acronym. It's an initialism because you pronounce each letter individually: H-T-T-P. An acronym is when a new word is created by letters. Since I know Felix likes space, NASA is an excellent example of an acronym. You pronounce it as one word, NASA (initialism), not as N-A-S-A (this would be an acronym).

So, because we're working with an initialism, we use `an` before a vowel sound, even if the first letter is a consonant. The pronunciation of H starts with "eh" (or "eg" in Felix's example), so we'd use `an`. "An XML file" is another example of a consonant having a vowel sound and thus requiring an `an`.

Reply:
TIL about initialisms vs acronyms. Thanks Brandon! Always fun to learn about language from you 🙂